src/style.css

```diff
@@ -28,13 +28,21 @@ body {
   align-items: center;
   border-bottom: 0.5px solid #000;
   font-style: italic;
+  padding: 20px 0;
+}
+
+.addtolist input {
+  width: 100%;
+  height: 25px;
 }
 
 .content {
   margin-top: 20px;
   display: flex;
   flex-direction: column;
   gap: 20px;
+  border-radius: 8px;
+  padding: 20px;
 }
 
 .row-item {
```

Comment:
Great job @PRATAP-KUMAR 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md), you may kindly implement the functionality to edit task descriptions